### DISCUSSION TITLE
Please welcome Andistorm as a new Collaborator!

### DISCUSSION THREAD

**Discussion by StuckAtPrototype** [StuckAtPrototype/Racer#32] [Announcements]
Hi everyone! Please give a warm welcome to @andistorm. He is now officially a collaborator on the Racer and Thumbtroller projects. Thanks so much Andistorm for taking on this task and I look forward to the collaboration. Welcome aboard. 

I look forward to see where we can take this project as a community, thank you everyone and thank you Andistorm! 

Cheers to the future. Lets build cool stuff together!

**Comment by andistorm** [comment]
Hi everyone! Thank you for the warm welcome! I’m excited to join the Racer and Thumbtroller projects and can’t wait to collaborate with all of you. I look forward to seeing what we can achieve together as a community. Let's start to build some amazing things!

Cheers! 🐧
